lib/dogma.ex

```diff
@@ -10,6 +10,8 @@ defmodule Dogma do
   alias Dogma.Rules
   alias Dogma.ScriptSources
 
+  @version Mix.Project.config[:version]
```

Comment:
Won't this return the version of the project that requires Dogma, not Dogma itself?